### PR TITLE
Closes Issue 1

### PR DESCRIPTION
**Pull request by madison-kerndt**
## Purpose

This pull request passes the first test provided to us in the first Issue of the Remember app. The end result is the the list items, provided by mirage, display on the home page (at the root directory). 

## Approach

This change now renders content onto the front page of the application. 

### Learning

We fumbled linking all of the various elements. In the end we asked instructors, classmates and referred to the sample github repository. 

https://github.com/turingschool-examples/ember-groceries

### Open Questions and Pre-Merge TODOs

May need to add more thorough tests of this functionality. Other than that this pull should be pretty clean.  

### Test coverage 

Tests were provided, they test the existence of more than 5 items rendering on the front page of the application. 

### Follow-up tasks

- [Issue #2 (Example)](https://github.com/flexyford/pull-request/issues)

#### Before
Blank 

#### After

screen shot 2016-10-12 at 4 24 29 pm

### CODE REVIEW

**Comment by madison-kerndt** [comment]
@stevekinney @martensonbj

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('string'),
+  notes: DS.attr('string'),
+  pinned: DS.attr('boolean')
```
Comment (martensonbj) [inline app/models/reminder.js]: Pinned should default to false.

File: tests/unit/models/index-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test, skip } from 'ember-qunit';
+
+moduleForModel('index', 'Unit | Model | index', {
```
Comment (martensonbj) [inline tests/unit/models/index-test.js]: What is an index model? Was this a mistake? If you aren't using this model remove unnecessary files.

File: app/templates/reminders.hbs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<ul class="reminder">
+  {{#each model as |reminder|}}
+    <li class="spec-reminder-item">{{reminder.title}}</li>
```
Comment (martensonbj) [inline app/templates/reminders.hbs]: Interesting choice of HTML element. Eventually each of these will be autonomous reminders, should probably be rendered as `<section>` and `<article>`.

File: app/templates/index.hbs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<!-- <div class="reminder">
+  {{#each reminders as |reminder|}}
+    {{reminder-item reminder=reminder}}
+  {{/each}}
+</div>
+
+{{outlet}} -->
```
Comment (madison-kerndt) [inline app/templates/index.hbs]: Deleted, thank you!

File: tests/unit/models/index-test.js (hunk 2 above)
Comment (madison-kerndt) [inline tests/unit/models/index-test.js]: This was in the initial package, I think, or maybe added by mistake when creating a new ember element.

File: app/templates/reminders.hbs (hunk 3 above)
Comment (madison-kerndt) [inline app/templates/reminders.hbs]: Changed!

File: app/models/reminder.js (hunk 1 above)
Comment (martensonbj) [inline app/models/reminder.js]: Yep! You can add that default as a property of the model directly. Like [this grocery model](https://github.com/turingschool-examples/ember-groceries/blob/master/app/models/grocery.js#L8)

Comment (stevekinney) [inline app/models/reminder.js]: Check out the documentation here: https://guides.emberjs.com/v2.4.0/models/defining-models/#toc_options

File: app/templates/index.hbs (hunk 4 above)
Comment (stevekinney) [inline app/templates/index.hbs]: Did you push up this change because it still appears to be in the PR.

**Comment by stevekinney** [comment]
My suspicion is that you pushed changes to a new PR (#3) instead of this one.

**Comment by madison-kerndt** [comment]
@martensonbj @stevekinney I figured out to submit new changes to an old pull request, could you review it?

**Comment by martensonbj** [comment]
Looks good!

**Comment by stevekinney** [comment]
In the future, it would be better to name this PR after the feature it implements in addition to the issue it closes.

**Pull request merged**